faculty_cli/cli.py

```diff
@@ -1451,3 +1450,148 @@ def dataset_ls(project, prefix, show_hidden):
         prefix, project_id=project_id, show_hidden=show_hidden
     ):
         click.echo(item)
+
+
+@cli.group()
+def template():
+    """Manipulate templates in the Faculty knowledge centre."""
+    pass
+
+
+@template.command()
+def init():
+    """Create a blank template."""
+    print("hello")
+
+
+@template.command()
+@click.argument(
+    "source_template_name"
```

Comment:
For cleanness / consistency with other commands, would recommend:
```suggestion
    "template"
```

Reply:
Done